lib/github-pages.rb

```diff
@@ -7,29 +7,31 @@ class GitHubPages
   def self.gems
     {
       # Jekyll
-      "jekyll"                => "2.4.0",
-      "jekyll-coffeescript"   => "1.0.1",
+      "jekyll"                => "3.0.0",
       "jekyll-sass-converter" => "1.3.0",
```

Comment:
@benbalter Do you mean why we removed coffeescript and not sass? Two reasons: Sass is very heavily used, whereas CoffeeScript is muuuch less-oft used, and because coffeescript had a hard dependency on NodeJS/therubyracer (see https://github.com/jekyll/jekyll/issues/3324#issuecomment-72342813). So we opted to keep sass-converter because we felt it was used often and it didn't require many extra dependencies so it was a huge win there, too.